docs/API.md

```diff
@@ -717,6 +717,7 @@ __Parameters__
 |`objectName` | _string_ | Name of the object. |
 |`expiry`     | _number_ | Expiry time in seconds. Default value is 7 days. |
 |`reqParams`  | _object_ | request parameters. |
+|`requestDate`  | _Date_ | A date object, the url will be issued at. Default value is now. |
```

Comment:
`(optinal)` information should be added to the above `expiry`, `reqParams`, and `requestDate` descriptions.